README.md

```diff
@@ -126,38 +126,35 @@ Whenever you want to update the github related information on the website you ne
 Note here that if you click [my portfolio](https://ashutosh1919.github.io), you can see animating the logo at the beginning. I have designed that logo in [`Figma`](https://www.figma.com/) and then animated it using css.
 Therefore, this part of portfolio is not customizable. But don't worry we have a solution to this problem. You have below two alternatives:
 
-- If you want to design your own logo, then you can design it using `Figma` or `Adobe XD` or `Adobe Illustrator` or `Inkscape`. If you want to animate it, you can refer to `./src/components/Loader` directory which contains `js` and `css` files which animates the logo.
-- If you don't want Splash screen or you don't know how to design logo, then this option is for you.
+- If you want to design your own logo, then you can design it using `Figma` or `Adobe XD` or `Adobe Illustrator` or `Inkscape`. If you want to animate it, you can refer to `./src/components/Loader` directory which contains `js` and `css` files which animates the logo
+- If you don't want Splash screen or you don't know how to design logo, then this option is for you
```

Comment:
```suggestion
- If you don't want Splash screen or you don't know how to design logo, then this option is for you.
```

Reply:
Bullet points should not end with a punctuation, expect when its the last point